region/crateria/west/Gauntlet Entrance.json

```diff
@@ -394,6 +394,35 @@
         "Breaking the center blocks opens up a runway that can be used to charge a new spark in room."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Come In With Blue Spring Ball Bounce",
+      "entranceCondition": {
+        "comeInWithBlueSpringBallBounce": {
+          "movementType": "controlled"
+        }
+      },
+      "requires": [
+        "canSlowShortCharge"
+      ],
+      "clearsObstacles": ["A"]
+    },
+    {
+      "link": [1, 2],
+      "name": "Come In Getting Blue Speed, Spring Ball Bounce",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 6,
+          "openEnd": 1,
+          "steepUpTiles": 1,
+          "steepDownTiles": 1
+        }
+      },
+      "requires": [
+        "canSlowShortCharge"
```

Comment:
- Springball and speedball tech
- trickyJump to each, but the `canSlowShortCharge` places it better at expert.